src/Http/Request/RequestExecutor.ts

```diff
@@ -394,4 +449,24 @@ export class RequestExecutor extends Observable implements IRequestExecutor {
 
     this._faildedNodesStatuses.clear();
   }
+
+  protected unauthorizedError(serverNode: ServerNode, command: RavenCommand): AuthorizationException {
+    let message: string = null;
+
+    if (!!serverNode.database) {
+      message = `database ${serverNode.database} on `;
+    }
+
+    message = `Forbidden access to ${message}${serverNode.url} node, `;
+
+    if (this._authOptions.certificate) {
+      message = `${message}certificate does not have permission to access it or is unknown.`;
+    } else {
+      message = `${message}a certificate is required.`;
+    }
+
+    message = `${message} ${command.method} ${command.pathWithNode(serverNode)}`;
+
+    return new AuthorizationException(message);
```

Comment:
Can we get an actual SSL error in here somehow?

Reply:
I think, yes